hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/transfer/ClassicTransfersCall.java

```diff
@@ -232,9 +231,7 @@ private static long minimumTinybarPriceGiven(
         final Set<Bytes> aliasesToLazyCreate = new HashSet<>();
         for (final var tokenTransfers : op.tokenTransfers()) {
             final var unitAdjusts = tokenTransfers.transfers();
-            // (FUTURE) Remove this divisor special case, done only for mono-service fidelity
-            final var sizeDivisor = Arrays.equals(selector, TRANSFER_TOKEN.selector()) ? 2 : 1;
-            minimumTinybarPrice += (unitAdjusts.size() / sizeDivisor) * baseUnitAdjustTinybarPrice;
+            minimumTinybarPrice += unitAdjusts.size() * baseUnitAdjustTinybarPrice;
```

Comment:
(There's something behind this, was it just a mistake in mono? Curious...)

Reply:
I _think_ this has to do with the fact that because a transfer involves 2 legs we were adjusting for this here.  This fact is accounted for in the `transferGasRequirement` method